src/main/java/com/mitchej123/hodgepodge/asm/transformers/mc/PlayerManagerTransformer.java

```diff
@@ -14,53 +14,51 @@
 import org.objectweb.asm.tree.MethodNode;
 import org.objectweb.asm.tree.TypeInsnNode;
 
+import com.mitchej123.hodgepodge.asm.HodgepodgeClassDump;
+import com.mitchej123.hodgepodge.core.HodgepodgeCore;
+
 @SuppressWarnings("unused")
 public class PlayerManagerTransformer implements IClassTransformer {
 
-    private static final String TARGET_CLASS = "net.minecraft.server.management.PlayerManager";
-    private static final String METHOD_NAME = "filterChunkLoadQueue";
-    private static final String[] METHOD_NAMES_OBF = { "func_72691_b", "b" };
-
     @Override
     public byte[] transform(String name, String transformedName, byte[] basicClass) {
-        if (!TARGET_CLASS.equals(transformedName)) {
-            return basicClass;
+        if (basicClass == null) return null;
+        if ("net.minecraft.server.management.PlayerManager".equals(transformedName)) {
+            final byte[] transformedBytes = transformBytes(basicClass);
+            HodgepodgeClassDump.dumpClass(transformedName, basicClass, transformedBytes, this);
+            return transformedBytes;
         }
+        return basicClass;
+    }
 
+    private static byte[] transformBytes(byte[] basicClass) {
         ClassReader classReader = new ClassReader(basicClass);
         ClassNode classNode = new ClassNode();
         classReader.accept(classNode, 0);
-
         boolean changed = false;
-
         for (MethodNode method : classNode.methods) {
             if (isTargetMethod(method)) {
                 changed |= transformMethod(method);
             }
         }
-
         if (changed) {
             ClassWriter classWriter = new ClassWriter(ClassWriter.COMPUTE_MAXS | ClassWriter.COMPUTE_FRAMES);
             classNode.accept(classWriter);
             return classWriter.toByteArray();
         }
-
         return basicClass;
     }
 
-    private boolean isTargetMethod(MethodNode method) {
-        if (METHOD_NAME.equals(method.name)) {
-            return true;
-        }
-        for (String obfName : METHOD_NAMES_OBF) {
-            if (obfName.equals(method.name)) {
-                return true;
-            }
+    private static boolean isTargetMethod(MethodNode method) {
+        if (HodgepodgeCore.isObf()) {
```

Comment:
I've seen issues with this being cached and the transformer running both before and after the deobf transformer, that's why a lot of transformers I've written (including Angelica) check both obf and deobf.

Reply:
if your transformer runs both before and after the deobf transformer you have a serious problem because your transformer is registered multiple times at different priority index